checks.xml

```diff
@@ -86,7 +86,7 @@
             <property name="allowNonPrintableEscapes" value="true"/>
         </module>
         <module name="LineLength">
-            <property name="max" value="100"/>
```

Comment:
This should probably be considered in another PR.

Reply:
In my code I use 120, is it really important to extract it in another PR?